### PR TITLE
Updated the existing documentation

### PR DESCRIPTION
**Pull request by sarthak2007**
### Description

I have updated the documentation in order to understand how to use the updated coordinate systems.

### Tasklist

- [x] Ensure all CI builds pass
- [x] Review and approve

### CODE REVIEW

File: doc/tutorials/coordinate/astronomical_coordinate.md [hunk 1]
```diff
@@ -18,23 +18,38 @@ Supported coordinate systems:
...
 
 int main()
 {
     //first we create a point we want to represent in the sky
-    spherical_representation star(30.455, 85.56);
+    auto star = make_spherical_representation(45.0 * bud::degrees, 45.0 * bud::degrees, 3.0 * meters);
 
     //we define motion it has (if it has)
-    spherical_coslat_differential motion(0.01,0.03);
+    auto motion = make_spherical_coslat_differential(10.0 * bud::degrees, 15.0 * bud::degrees, 1.0 * meters);
```
Comment (Ruchika2696) [inline doc/tutorials/coordinate/astronomical_coordinate.md]: use velocity unit

File: doc/tutorials/coordinate/astronomical_coordinate_operation.md [hunk 2]
```diff
@@ -5,42 +5,61 @@ There are only a few operations available at this time on coordinate which are d
...
 
 using namespace boost::astronomy::coordinate;
+using namespace boost::units;
+using namespace boost::units::si;
+namespace bud = boost::units::degree;
+
+typedef spherical_representation<double, quantity<degree::plane_angle>, quantity<degree::plane_angle>,
+        quantity<si::length>> spherical_rep;
+typedef spherical_coslat_differential<double, quantity<degree::plane_angle>,
+        quantity<degree::plane_angle>, quantity<si::length>> spherical_coslat_diff;
```
Comment (Ruchika2696) [inline doc/tutorials/coordinate/astronomical_coordinate_operation.md]: change it to velocity unit in differential

File: doc/tutorials/coordinate/astronomical_coordinate_operation.md [hunk 3]
```diff
@@ -5,42 +5,61 @@ There are only a few operations available at this time on coordinate which are d
...
-        //first we create a point we want to represent in the sky
-    spherical_representation star(30.455, 85.56);
+    //first we create a point we want to represent in the sky
+    auto star1 = make_spherical_representation(45.0 * bud::degrees, 45.0 * bud::degrees, 3.0 * meters);
+    auto star2 = make_spherical_representation(15.0 * bud::degrees, 5.0 * bud::degrees, 5.0 * meters);
 
     //we define motion it has (if it has)
-    spherical_coslat_differential motion(0.01,0.03);
+    auto motion1 = make_spherical_coslat_differential(10.0 * bud::degrees, 15.0 * bud::degrees, 1.0 * meters);
+    auto motion2 = make_spherical_coslat_differential(70.0 * bud::degrees, 1.5 * bud::degrees, 10.0 * meters);
```
Comment (Ruchika2696) [inline doc/tutorials/coordinate/astronomical_coordinate_operation.md]: change it to velocity unit in differential

File: doc/tutorials/coordinate/coordinate_point.md [hunk 4]
```diff
@@ -41,40 +41,42 @@ or
...
 
     //creating a point from another point
-    cartesian point3(point2); //or point3 = point2
+    auto point3 = make_cartesian_representation(point2);
 
     //creating a point from boost::geometry::model::point
     //any type of point can be used here 
     //here we will demonstrate with cartesian point
     //https://www.boost.org/doc/libs/1_67_0/libs/geometry/doc/html/geometry/reference/models/model_point.html
     //https://www.boost.org/doc/libs/1_60_0/libs/geometry/doc/html/geometry/reference/cs.html
```
Comment (Ruchika2696) [inline doc/tutorials/coordinate/coordinate_point.md]: these links should be from the latest stable version currently available.

File: doc/tutorials/coordinate/coordinate_point.md [hunk 5]
```diff
@@ -109,41 +112,44 @@ or
...
 
     //creating a point from another point
-    spherical<boost::astronomy::coordinate::radian> point3(point2); //or point3 = point2
+    auto point3 = make_spherical_representation(point2);
 
     //creating a point from boost::geometry::model::point
     //any type of point can be used here 
     //here we will demonstrate with cartesian point
     //https://www.boost.org/doc/libs/1_67_0/libs/geometry/doc/html/geometry/reference/models/model_point.html
     //https://www.boost.org/doc/libs/1_60_0/libs/geometry/doc/html/geometry/reference/cs.html
```
Comment (Ruchika2696) [inline doc/tutorials/coordinate/coordinate_point.md]: these links should be from the latest stable version currently available.

File: doc/tutorials/coordinate/vector_operation.md [hunk 6]
```diff
@@ -13,34 +13,45 @@ Representation classes have provided a few of the most used vector operations wh
...
-    cartesian_representation unit_vector = point2.unit_vector<cartesian_representation>();
+    auto unit_vector_point = unit_vector(point1);
 
     //magnitude
-    std::cout << point1.magnitude() << std::endl;
+    std::cout << magnitude(point1) << std::endl;
 
     //sum of two vectors
     //Here we show the flexibility provided by the library to use any representation for the operations and implicit casting
     //right hand of the equal returns object of cartesian_representation
```
Comment (Ruchika2696) [inline doc/tutorials/coordinate/vector_operation.md]: change this to "right-hand side"

File: doc/tutorials/coordinate/vector_operation.md [hunk 7]
```diff
@@ -13,34 +13,45 @@ Representation classes have provided a few of the most used vector operations wh
...
-    //But it gets converted implicitly into spherical_representation<radian>
-    spherical_representation<radian> sum_result = point1.sum<cartesian_representation>(point2);
+    //But it gets converted implicitly into spherical_representation
```
Comment (Ruchika2696) [inline doc/tutorials/coordinate/vector_operation.md:48]: I doubt this. have you tried running this code?

**Review by Ruchika2696 — CHANGES_REQUESTED** [review]

File: doc/tutorials/coordinate/vector_operation.md (hunk 7 above)
Comment (sarthak2007) [reply doc/tutorials/coordinate/vector_operation.md:48]: Yes it executed successfully

**Review by Ruchika2696 — APPROVED** [review]

**Pull request merged**